### DISCUSSION TITLE
How to obtain commandline args from HostService in Console Template?

### DISCUSSION THREAD

**Discussion by RayMMond** [abpframework/abp#11401] [Q&A] [answered]
How to obtain commandline args from `HostService` in Console Template?

**Comment by EngincanV** [comment, marked as answer]
You can use the [`Environment.GetCommandLineArgs()`](https://docs.microsoft.com/en-us/dotnet/api/system.environment.getcommandlineargs?view=net-6.0) method:

```csharp
using System;

string[] args = Environment.GetCommandLineArgs();
```

**Reply by RayMMond** [reply]
😄 Thanks~
